sops_test.go

```diff
@@ -116,6 +116,10 @@ func TestEncrypt(t *testing.T) {
 			Key:   "foobar",
 			Value: 2.12,
 		},
+		TreeItem{
```

Comment:
Could you add the same thing to `TestDecrypt`? We should test that it works both ways.

Reply:
Added! Now it tests the value when is decrypting